metaflow/plugins/cards/ui/package.json

```diff
@@ -9,35 +9,40 @@
     "check": "svelte-check --workspace src",
     "lint": "eslint src --ext .ts,.svelte",
     "lint:fix": "yarn run lint --fix",
-    "prebuild": "yarn run check && yarn run lint"
+    "prebuild": "yarn run check && yarn run lint",
+    "cypress:open": "wait-on http://localhost:8080 && cypress open",
```

Comment:
Can you get this to run `yarn dev` automatically if it is not running? It took me a little while to work out why the tests weren't running

Reply:
that was my initial implementation as well.  However, Cypress specifically advises not to do that here:

https://docs.cypress.io/guides/continuous-integration/introduction#Boot-your-server

We are definitely going to need come up with a different way to test the output for build time.  so my hope is that we'll automate all of this away, and for local dev, you can just keep the separate terminal window open.